recipes/gklib/meta.yaml

```diff
@@ -0,0 +1,93 @@
+  build:
+    - {{ compiler('c') }}
+    - cmake >=3.15
+    - llvm-openmp  # [osx]
+    - libgomp  # [linux]
+
+test:
+  commands:
+
+    - ${PREFIX}/bin/cmpnbrs --help  # [unix]
+    - ${PREFIX}/bin/csrcnv --help  # [unix]
+    - ${PREFIX}/bin/fis --help  # [unix]
+    - ${PREFIX}/bin/gkgraph --help  # [unix]
+    - ${PREFIX}/bin/gkrw --help  # [unix]
+    - ${PREFIX}/bin/m2mnbrs --help  # [unix]
+    - test -f ${PREFIX}/include/gk_arch.h  # [unix]
+    - test -f ${PREFIX}/include/gk_defs.h  # [unix]
+    - test -f ${PREFIX}/include/gk_externs.h  # [unix]
+    - test -f ${PREFIX}/include/gk_getopt.h  # [unix]
+    - test -f ${PREFIX}/include/GKlib.h  # [unix]
+    - test -f ${PREFIX}/include/gk_macros.h  # [unix]
+    - test -f ${PREFIX}/include/gk_mkblas.h  # [unix]
+    - test -f ${PREFIX}/include/gk_mkmemory.h  # [unix]
+    - test -f ${PREFIX}/include/gk_mkpqueue2.h  # [unix]
+    - test -f ${PREFIX}/include/gk_mkpqueue.h  # [unix]
+    - test -f ${PREFIX}/include/gk_mkrandom.h  # [unix]
+    - test -f ${PREFIX}/include/gk_mksort.h  # [unix]
+    - test -f ${PREFIX}/include/gk_mkutils.h  # [unix]
+    - test -f ${PREFIX}/include/gk_proto.h  # [unix]
+    - test -f ${PREFIX}/include/gkregex.h  # [unix]
+    - test -f ${PREFIX}/include/gk_struct.h  # [unix]
+    - test -f ${PREFIX}/include/gk_types.h  # [unix]
+    - test -f ${PREFIX}/include/ms_inttypes.h  # [unix]
+    - test -f ${PREFIX}/include/ms_stat.h  # [unix]
+    - test -f ${PREFIX}/include/ms_stdint.h  # [unix]
+    - test -f ${PREFIX}/lib/libGKlib${SHLIB_EXT}  # [unix]
+
+    - if not exist "%LIBRARY_INC%\\gk_arch.h" exit 1  # [win]
+    - if not exist "%LIBRARY_INC%\\gk_defs.h" exit 1  # [win]
+    - if not exist "%LIBRARY_INC%\\gk_externs.h" exit 1  # [win]
+    - if not exist "%LIBRARY_INC%\\gk_getopt.h" exit 1  # [win]
+    - if not exist "%LIBRARY_INC%\\GKlib.h" exit 1  # [win]
+    - if not exist "%LIBRARY_INC%\\gk_macros.h" exit 1  # [win]
+    - if not exist "%LIBRARY_INC%\\gk_mkblas.h" exit 1  # [win]
+    - if not exist "%LIBRARY_INC%\\gk_mkmemory.h" exit 1  # [win]
+    - if not exist "%LIBRARY_INC%\\gk_mkpqueue2.h" exit 1  # [win]
+    - if not exist "%LIBRARY_INC%\\gk_mkpqueue.h" exit 1  # [win]
+    - if not exist "%LIBRARY_INC%\\gk_mkrandom.h" exit 1  # [win]
+    - if not exist "%LIBRARY_INC%\\gk_mksort.h" exit 1  # [win]
+    - if not exist "%LIBRARY_INC%\\gk_mkutils.h" exit 1  # [win]
+    - if not exist "%LIBRARY_INC%\\gk_proto.h" exit 1  # [win]
+    - if not exist "%LIBRARY_INC%\\gkregex.h" exit 1  # [win]
+    - if not exist "%LIBRARY_INC%\\gk_struct.h" exit 1  # [win]
+    - if not exist "%LIBRARY_INC%\\gk_types.h" exit 1  # [win]
+    - if not exist "%LIBRARY_INC%\\ms_inttypes.h" exit 1  # [win]
+    - if not exist "%LIBRARY_INC%\\ms_stat.h" exit 1  # [win]
+    - if not exist "%LIBRARY_INC%\\ms_stdint.h" exit 1  # [win]
+    - if not exist "%LIBRARY_LIB%\\GKlib.lib" exit 1  # [win]
+    - if not exist "%LIBRARY_BIN%\\cmpnbrs.exe" exit 1  # [win]
```

Comment:
Can these be executed with --help as well?

Reply:
Apparently they can! Fixed by 38bf7b6.